### PR TITLE
tools: link to more web services

### PR DESCRIPTION
**Pull request by warpfork**
We've got all this stuff out there, and nobody knows about it :)

(Including me!  Most of these just came from discussion in the ipld community chat!  I decided to write them down before they slip out of working memory again.)

### CODE REVIEW

**Comment by rvagg** [comment]
FYI during the time this PR has been up, the beginnings of some neat CAR functionality has been added to explore.ipld.io by @alvin-reyes (ongoing work, more stuff going in)

**Review by RangerMauve — APPROVED** [review]
I think this looks good even without the car.ipfs.io (since it doesn't even mention IPLD. 😅)

File: tools/index.md [hunk 1]
```diff
@@ -19,11 +20,41 @@ It contains basic operations like `ipld read` (which includes the ability to do
...
-(This needs no introduction ;)  Just look at it!)
+The IPLD Explorer loads and visualizes IPLD data, and lets you navigate it interactively.
+
+Some well-known example datasets are included in the front page, which you can navigate to get a quick idea of what IPLD is all about.
+
+Some datasets using unixfs are specially treated, and can be navigated by the filesystem-like abstraction offered by unixfs.
+(In the future, we hope to make these features more generalized, by the use of ADLs; at present, unixfs's special behaviors are hardcoded.)
+
+Note that to be navigatable in the IPLD Explorer, the data has to be found somewhere in the public IPFS network swarm.
+(Code contributions to increase the variety of ways data can be supplied to the IPLD Explorer would be welcome!)
```
Comment (rvagg) [inline tools/index.md:36]:
```suggestion
Note that to be navigatable in the IPLD Explorer, the data has to be found somewhere in the public IPFS network swarm. Alternatively, a CAR file containing a DAG can be uploaded directly to the explorer for inspection.
(Code contributions to increase the variety of ways data can be supplied to the IPLD Explorer would be welcome!)
```

**Comment by rvagg** [comment]
going to merge this as is so we can get it off the table; someone else can add car.ipfs.io as a separate PR

**Pull request merged**